docs/design/launch_system.md

```diff
@@ -22,20 +24,27 @@ Each folder contains a `launch/` folder where all the launch files should be pla
 
 ### File Naming Convention
 
-All launch files end in `.launch.py`. Furthermore, all launch files specific to the a vehicle platform or orchastrator launch files are prefixed with `art_`.
+All launch files end in `.launch.py`. Furthermore, all launch files specific to a vehicle platform or orchastrator launch files are prefixed with `art_`.
 
 ## `art_launch/`
 
-This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). This file will do a few things.
+This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). This is a common launch file through which most services will go through. `art_launch` accomplishes two tasks:
 
-1. It will first define system wide parameters (e.g. `LaunchConfigurations`, `LaunchDescriptions`, etc.).
-2. It will create a [composable node container](https://docs.ros.org/en/galactic/How-To-Guides/Launching-composable-nodes.html).
-3. It will include all other orchestration launch files.
+1. It will create a [composable node container](https://docs.ros.org/en/galactic/How-To-Guides/Launching-composable-nodes.html).
+2. It will include all other orchestration launch files.
+
+## `art_dev_launch/`
+
+The purpose of this file is to define system wide parameters (e.g. `LaunchConfigurations`, `LaunchDescriptions`, etc.) for the `dev` service. This will allow all orchestrators to know which specific packages to launch, and which to ignore. This is currently set up to run a cone-based path following demo, but can be used to test any other packages.
```

Comment:
is this true? are those system wide parameters included by the vehicle specific launches or are they copied?

Reply:
I'm not sure what you mean here; if the vehicle has the exact same intended task, then yes the params are just copied. If a certain package is intended to be launched in the exact same way, then yes it is just copied (e.g. both services want "EKF"). But in my mind two vehicles / services don't necessarily share any parameters, so they should each define them themselves.

Reply:
maybe some of the "use_sim parameters would make sense to be shared, but even those I could see some scenarios where they shouldn't be...